test/x3/constexpr.cpp

```diff
@@ -0,0 +1,75 @@
+/*=============================================================================
+    Copyright (c) 2015 Joel de Guzman
+
+    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+=============================================================================*/
+#include <boost/detail/lightweight_test.hpp>
+
+#define BOOST_SPIRIT_X3_DISABLE_SIOF_CHECK
```

Comment:
This is left by omission, I guess.

Reply:
fixed in new commit [b7fc3bf](https://github.com/boostorg/spirit/pull/570/commits/b7fc3bf2eb0b5cfab6a79b9f07ac7930fba131ba)